manager_rethink.go

```diff
@@ -62,6 +62,24 @@ func rdbFromPolicy(p Policy) (*rdbSchema, error) {
 	}, err
 }
 
+// NewRethinkManager initializes a new RethinkManager for given session, table name defaults
+// to "policies", policies can default to nil.
+func NewRethinkManager(session *r.Session, table string, policies map[string]Policy) *RethinkManager {
```

Comment:
A sensible default, or empty is ok I think! gorethink will notify the user if an incorrect value was used on query.